docs(quickstart): update with changes from last release (#1488)
opstrace/opstrace: docs/quickstart.md

  * While GCP is also fully supported, we will focus on AWS in this quick start.
* For sending dummy data to Opstrace, you'll need [Docker](https://docs.docker.com/install) and [Docker Compose](https://docs.docker.com/compose/install)

> You will also need an Auth0 account to wire up Google or GitHub log in (sign up for free here).

Reviewer: Needs a link.

Author: Removing this line in favor of the authentication section below.

<!-- /tabs -->

If you see a proper help message, you're ready to go.

### Authentication

Opstrace is secure by default—it requires authentication (provided through Auth0) to work.
It's easy to get started—follow [this guide to set up Auth0](https://github.com/opstrace/opstrace/wiki/User-guide:-custom-Auth0-integration).

### DNS

> Next, create DNS records to point to your cluster by [following these instructions](https://github.com/opstrace/opstrace/wiki/User-guide:-Upgrade-Opstrace-with-a-custom-DNS-domain-and-custom-Auth0-integration).

Reviewer: ```suggestion
Next, set up a DNS domain for your Opstrace instance by [following these instructions](https://github.com/opstrace/opstrace/wiki/User-guide:-Upgrade-Opstrace-with-a-custom-DNS-domain-and-custom-Auth0-integration).
```

<!--/export-to-input-->

> The name will globally identify you in our domain as `$OPSTRACE_NAME.example.com`, which we provide for you by default as a convenience.

Reviewer: ```suggestion
The name will globally identify your Opstrace instance as `$OPSTRACE_NAME.example.com`.
```
